packages/react-components/src/components/Tag/Tag.module.scss

```diff
@@ -6,7 +6,6 @@
   border-radius: var(--radius-3);
   border-color: var(--border-basic-primary);
   background-color: var(--surface-tertiary-default);
-  overflow: hidden;
```

Comment:
This change will break the default behavior, it will break ellipsis in case of small container for tag

Reply:
@marcinsawicki Hi, can you please re-check it? Now tooltips are working differently than before, so maybe it would work properly now?